test/core/network/network_request_executor_test.dart

```diff
@@ -37,32 +37,32 @@ void main() {
 
   test('should return [ServerFailure] if api call fails', () async {
     // arrange
-    final tResponse = responseFromStatusCode(500, body: '');
+    final testResponse = responseFromStatusCode(500, body: '');
```

Comment:
i thought we agreed on some strategy to name these kinds of variables? I think the most important aspect is to remain consistent. Everywhere else i have used tResponse, tSomething as in 'the response', 'the something'

Reply:
I agree we should be consistent. In 727297093cab26834a29c9cfcb3d637c732f48b5, I changed all occurrences of `t[Name]` to `test[Name]`. We should generally strive to avoid abbreviations in code and this seemed like a quick fix.